mail_activity_team/__manifest__.py

```diff
@@ -11,11 +11,12 @@
     "author": "ForgeFlow, Sodexis, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "installable": True,
-    "depends": ["mail_activity_board"],
+    "depends": ["mail_activity_board", "web_domain_field"],
```

Comment:
This dependency is not something that is in v13, so it's not only forward-ports, and I would like to avoid new dependencies in the module.

Reply:
On 14+ it is not recommended to set a dynamic domain by the onchange. the only way I found was this :thinking: 

Maybe if odoo does a small change on SparseField we could do it, Changing this by https://github.com/odoo/odoo/blob/16.0/addons/base_sparse_field/models/fields.py#L89
```python
            return json.dumps(value) if isinstance(value, (dict, list)) else (value or None)
```

Might do the trick

Reply:
You can propose such change, but for this, you can still use onchanges. Being deprecated doesn't mean they are not allowed. But aren't you able to define a fixed domain not depending on content?

Reply:
You want to filter the users If and only If you have a team selected. We could try with a repeated field, who knows, that might work....

Reply:
I made something in order to avoid it. It would be interesting if someone can test it :smile:

Reply:
Please check how I have avoided the use of this module and still get good performance:

https://github.com/OCA/rma/pull/326